local_env/master/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jenkins/jenkins:2.375.1
+FROM jenkins/jenkins:jdk11
```

Comment:
I would prefer to use the `lts-jdk11` tag to keep it more stable.

Reply:
adapted as suggested